consensus/ibftlegacy/src/main/java/org/hyperledger/besu/consensus/ibftlegacy/IbftBlockHashing.java

```diff
@@ -133,33 +134,33 @@ private static BytesValue encodeExtraDataWithoutCommittedSeals(
 
     extraDataEncoding.endList();
 
-    return BytesValue.wrap(ibftExtraData.getVanityData(), extraDataEncoding.encoded());
+    return Bytes.wrap(ibftExtraData.getVanityData(), extraDataEncoding.encoded());
   }
 
-  private static BytesValue serializeHeader(
-      final BlockHeader header, final Supplier<BytesValue> extraDataSerializer) {
+  private static Bytes serializeHeader(
+      final BlockHeader header, final Supplier<Bytes> extraDataSerializer) {
     final BytesValueRLPOutput out = new BytesValueRLPOutput();
     out.startList();
 
-    out.writeBytesValue(header.getParentHash());
-    out.writeBytesValue(header.getOmmersHash());
-    out.writeBytesValue(header.getCoinbase());
-    out.writeBytesValue(header.getStateRoot());
-    out.writeBytesValue(header.getTransactionsRoot());
-    out.writeBytesValue(header.getReceiptsRoot());
-    out.writeBytesValue(header.getLogsBloom().getBytes());
-    out.writeUInt256Scalar(header.getDifficulty());
+    out.writeBytes(header.getParentHash());
+    out.writeBytes(header.getOmmersHash());
+    out.writeBytes(header.getCoinbase());
+    out.writeBytes(header.getStateRoot());
+    out.writeBytes(header.getTransactionsRoot());
+    out.writeBytes(header.getReceiptsRoot());
+    out.writeBytes(header.getLogsBloom());
```

Comment:
For type safety we will need to add back the `getBytes()`, or make LogsBloom a delegated bytes.   As long as it still can implement the plugins-api interface.  We may want to save that migration to a later PR however since the bytes inside is mutable and the API impact may be larger than a change in hierarchy.

Reply:
it's back now, sorry.